integreat_cms/cms/forms/pages/page_filter_form.py

```diff
@@ -100,3 +131,67 @@ def filter_by_translation_status(self, pages, language_slug):
             if translation_state in selected_status:
                 filtered_pages.append(page)
         return filtered_pages
+
+    def filter_by_latest_revision(self, pages, language_slug):
+        """
+        Filter the pages list by latest revision
+
+        :param pages: The list of pages
+        :type pages: list
+
+        :param language_slug: The slug of the current language
+        :type language_slug: str
+
+        :return: The filtered page list
+        :rtype: list
+        """
+        selected_status = self.cleaned_data["status"]
+        # Buffer variable because the pages list should not be modified during iteration
+        filtered_pages = []
+        for page in pages:
+            translation = page.get_translation(language_slug)
+            if translation.status == selected_status:
+                filtered_pages.append(page)
+        return filtered_pages
+
+    def filter_by_start_date(self, pages, language_slug):
+        """
+        Filter the pages list by start date
+
+        :param pages: The list of pages
+        :type pages: list
+
+        :param language_slug: The slug of the current language
+        :type language_slug: str
+
+        :return: The filtered page list
+        :rtype: list
+        """
+        selected_start_date = self.cleaned_data["date_from"]
+        filtered_pages = []
+        for page in pages:
+            translation = page.get_translation(language_slug)
+            if translation.last_updated.date() >= selected_start_date:
+                filtered_pages.append(page)
+        return filtered_pages
+
+    def filter_by_end_date(self, pages, language_slug):
+        """
+        Filter the pages list by end date
+
+        :param pages: The list of pages
+        :type pages: list
+
+        :param language_slug: The slug of the current language
+        :type language_slug: str
+
+        :return: The filtered page list
+        :rtype: list
+        """
+        selected_start_date = self.cleaned_data["date_from"]
```

Comment:
```suggestion
        selected_end_date = self.cleaned_data["date_to"]
```

Copy & Past problem?